docker/docker-compose.yml

```diff
@@ -34,6 +37,8 @@ services:
       - /phenix:/phenix:shared
       - ./tmp/minimega:/tmp/minimega
       - ./tmp/phenix:/tmp/phenix
+    environment:
+      MM_FILEPATH: /phenix/images
```

Comment:
Nothing in the core phenix code base uses this env variable, but I know the phenix-apps code base does. The phenix-apps code base defaults to this path... are you wanting to add this so folks know it's there to change? If so, we should probably add a comment for when/where it's used and when someone might want to change it (i.e., when it gets changed in the config for the minimega service).

Reply:
The latter, so people are aware of it, and also since it's specified for minimega. I'll add a comment.